packages/ui/Scroller/ScrollerBasic.js

```diff
@@ -340,6 +353,20 @@ class ScrollerBasic extends Component {
  * @public
  */
```

Comment:
Please check where is the right place for the documentation above.

Reply:
Yes for the same reason above. I should remove `moreInfo` though.